ci/launch_cluster.sh

```diff
@@ -29,33 +29,57 @@ VARIANT="$2"
 DEPLOYMENT_NAME="$3"
 
 if [ "$VARIANT" == "open" ]; then
-  TEMPLATE="https://s3.amazonaws.com/downloads.dcos.io/dcos/${CHANNEL}/cloudformation/multi-master.cloudformation.json"
+  INSTALLER_URL="https://downloads.dcos.io/dcos/${CHANNEL}/dcos_generate_config.sh"
 else
-  TEMPLATE="https://s3.amazonaws.com/downloads.mesosphere.io/dcos-enterprise-aws-advanced/${CHANNEL}/${VARIANT}/cloudformation/ee.multi-master.cloudformation.json"
+  if [ -z "$DCOS_INSTALLER_TEMPLATE" ]; then
+      echo "Expecting DCOS_INSTALLER_TEMPLATE environment variable"
+      exit 1
+  fi
+  if [ -z "$DCOS_LICENSE" ]; then
+      echo "Expecting $DCOS_LICENSE environment variable"
+      exit 1
+  fi
+  # Replace {} in the template with the channel
+  INSTALLER_URL=$(echo ${DCOS_INSTALLER_TEMPLATE} | sed "s|{}|${CHANNEL}|g")
```

Comment:
Why | and not / in sed?

Reply:
Because `/` will b present on the expanded channel 😉